src/main/java/com/cgdpd/library/service/BookService.java

```diff
@@ -30,6 +35,20 @@ public Book createBook(CreateBookRequestDTO requestDTO) {
         return bookMapper.mapToBook(createdBook);
     }
 
+    @Transactional(readOnly = true)
+    public PagedResponse<DetailedBookDTO> getBooks(PaginationCriteria paginationCriteria,
+                                                   SearchBookCriteria searchCriteria) {
+        var pageable = paginationCriteria.toPageRequest();
+        var books = bookRepository.findAll(
```

Comment:
We have different types of books, give it the accurate name
```suggestion
        var bookEntities = bookRepository.findAll(
```